tests/utils/test_exporter.py

```diff
@@ -779,6 +779,61 @@ def test_exporter_exports_requirements_txt_with_legacy_packages(tmp_dir, poetry)
     assert expected == content
 
 
+def test_exporter_exports_requirements_txt_with_legacy_packages_trusted_host(
+    tmp_dir, poetry
+):
+    poetry.pool.add_repository(LegacyRepository("custom", "http://example.com/simple",))
+    poetry.locker.mock_lock_data(
```

Comment:
You can drop most of the mock; effectively a no package data is fine since we are only testing the the indices. No point in duplicating data here.

Reply:
I remove a package from the mock data, think I can't remove other stuff or the test will fail...